--- xtables-addons-1.14/extensions/libxt_ipp2p.c.orig	2009-03-31 22:55:29.000000000 +0200
+++ xtables-addons-1.14/extensions/libxt_ipp2p.c	2009-04-10 15:13:50.000000000 +0200
@@ -17,7 +17,7 @@
 #include <ctype.h>
 #include <xtables.h>
 #include "xt_ipp2p.h"
-#define param_act(t, s, f) param_act((t), "ipp2p", (s), (f))
+#define xtables_param_act(t, s, f) xtables_param_act((t), "ipp2p", (s), (f))
 
 static void ipp2p_mt_help(void)
 {
@@ -103,8 +103,8 @@ static int ipp2p_mt_parse(int c, char **
 		break;
 
 	case 'b':		/*cmd: bit*/
-		xtables_param_act(XTF_ONLY_ONCE, "--kazaa", *flags & IPP2P_BIT);
-		xtables_param_act(XTF_NO_INVERT, "--kazaa", invert);
+		xtables_param_act(XTF_ONLY_ONCE, "--bit", *flags & IPP2P_BIT);
+		xtables_param_act(XTF_NO_INVERT, "--bit", invert);
 		*flags    |= IPP2P_BIT;
 		info->cmd |= IPP2P_BIT;
 		break;
